.env.docker

```
@@ -549,3 +549,42 @@ MCP_WS_SESSION_COOKIE_NAME=mcp-ws-session-id
+MCP_AUTH_SERVERS=http://localhost:3000
+
+# JWT validation for docker development (using RS256 with public key)
+MCP_AUTH_JWT_AUDIENCE=http://localhost:3001
+MCP_AUTH_JWT_ISSUER=http://localhost:3000
+MCP_AUTH_JWT_ALGORITHMS=RS256,ES256
+
+# Option 1: Use JWKS URI for dynamic key discovery (recommended)
+MCP_AUTH_JWT_JWKS_URI=http://localhost:3000/.well-known/jwks.json
+
+# Docker development authorization server configuration
+MCP_AUTH_SERVERS='[{
```

dotenv-linter (3.3.0)

[warning] [DuplicatedKey] The MCP_AUTH_SERVERS key is duplicated

(DuplicatedKey)